bower.json

```diff
@@ -16,6 +17,7 @@
     "pickadate": "3.5.4",
     "angular-ui-tree": "2.4.0",
     "ccp-js": "https://ccp.ccci.org/js/ccp-js.zip",
+    "cru-payments": "https://registry.npmjs.org/cru-payments/-/cru-payments-1.1.0.tgz",
```

Comment:
Can you update this to [`1.1.1`](https://github.com/CruPCI/cru-payments/releases/tag/1.1.1)?  I added American Express CVV validation. I probably should have mentioned it earlier but I was trying not to spam you with really small updates. I didn't know if there'd be more.